docs/components/generic/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Generic Component"
+linkTitle: "Generic"
+childTitleEndOverwrite: "Generic Component"
+weight: 55
+type: "docs"
+description: "Use a generic component to implement a component that does not fit any of the other APIs."
+tags: ["generic", "components"]
+icon: "/icons/components.png"
+images: ["/icons/components.png"]
+no_list: true
+# SMEs:
+---
+
+The _generic_ component type is for custom components that do not fit into any of the other component {{< glossary_tooltip term_id="subtype" text="subtypes" >}}.
+
+There are no built-in generic component models (other than `fake`).
+Generic is only for [modular resource](/extend/modular-resources/) models that don't fit into any of the typical categories.
+
+Use cases:
+
+- You want to implement a new model of component as a modular resource, and your device isn't an arm, a base, a board, a camera, or any of the other [built-in Viam component types](/components/).
```

Comment:
You say this three times (line 15, 18, and 22) - please change this intro paragraph and condense that to once

Reply:
Okay I reworked a bit--still says it 1.5 times but one is about API and one is about hardware. I kinda want to say it a couple of different ways since subtypes can be so confusing for people new to Viam. LMK if you have suggestions.